<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>820</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icon/free-icon-hand-cursor-5781782.png</normaloff>:/icon/free-icon-hand-cursor-5781782.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>-450</x>
      <y>-40</y>
      <width>2401</width>
      <height>900</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">

background-image: url(:/icon/Screenshot from 2024-07-04 10-28-03.png);
    background-position: bottom center; /* 바닥 중앙에서 시작 */
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>630</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Dyuthi</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Eat Here</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>630</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Dyuthi</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Take Out</string>
    </property>
   </widget>
   <widget class="QPushButton" name="eat_here_Btn">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>170</y>
      <width>381</width>
      <height>431</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/icon/free-icon-bar-6028207.png);
background-color: rgb(255, 255, 255);
; border-radius: 50; border: 2px solid

</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="takeout_Btn">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>170</y>
      <width>381</width>
      <height>431</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/icon/free-icon-takeaway-7770713.png);
background-color: rgb(255, 255, 255);
; border-radius: 50; border: 2px solid
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_back">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>70</y>
      <width>371</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Dyuthi</family>
      <pointsize>35</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 0, 0, 0); /* 투명 배경 */</string>
    </property>
    <property name="text">
     <string>To Go? Or Eat Here?</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resource_serve.qrc"/>
 </resources>
 <connections/>
</ui>
